r/R/duckdb.R

```diff
@@ -57,6 +57,10 @@ to_duckdb <- function(.data,
                       table_name = unique_arrow_tablename(),
                       auto_disconnect = TRUE) {
   .data <- as_adq(.data)
+  if (!requireNamespace("duckdb", quietly = TRUE)) {
+    abort("Please install the `duckdb` package.")
```

Comment:
Could we add a bit about the resaon why one might need to install duckdb? Something like "The `duckdb` package is required to pass data with `to_duckdb()`" or "Please install the `duckdb` package to pass data with `to_duckdb()`"